### DISCUSSION TITLE
Using MIT-licensed packages

### DISCUSSION THREAD

**Discussion by cloudcompute** [documenso/documenso#1293] [General]
While the Apps (in the apps/ directory) are AGPL licensed, the Packages (residing in the packages/ directory) are MIT licensed. 

Since the packages are MIT licensed, can we use some of these packages in our source code?

Thanks

**Comment by Mythie** [comment]
Hey there that's the intention! 

We want to leave some code for people to learn from and use without worrying about licensing but keep the main apps and a few other areas under less permissive licenses so we can keep Documenso running smoothly.

**Comment by cloudcompute** [comment]
@Mythie

Thanks for the clarification.
